Source/ACE.Server/WorldObjects/Creature_Death.cs

```diff
@@ -361,10 +361,10 @@ public void DoCantripLogging(WorldObject killer, WorldObject wo)
             var legendaryCantrips = wo.LegendaryCantrips;
 
             if (epicCantrips.Count > 0)
-                log.Info($"[EPIC] {Name} ({Guid}) generated item with {epicCantrips.Count} epic{(epicCantrips.Count > 1 ? "s" : "")} - {wo.Name} ({wo.Guid}) - {GetSpellList(epicCantrips)} - killed by {killer.Name} ({killer.Guid})");
+                log.Debug($"[EPIC] {Name} ({Guid}) generated item with {epicCantrips.Count} epic{(epicCantrips.Count > 1 ? "s" : "")} - {wo.Name} ({wo.Guid}) - {GetSpellList(epicCantrips)} - killed by {killer.Name} ({killer.Guid})");
```

Comment:
most definitely not debug messages.

Reply:
Don't get caught up in the term "debug".

What you should really be asking is, is this a message that should go to the console output during normal server operation?

In ACE, the way we define that threshold is between Debug and Info log levels. Info goes to the console, Debug does not.

Reply:
in that case, yes... if the server admin chooses to enable epic info, or any particular info such as corpse logging, then all that info should be displayed on the server console

in fact, i'm not even sure i understand why LogLevel.Debug wouldn't be displayed on the server console. i think of Debug as something only devs would enable mostly, and this would show information such as 'player is moving from landblock XXYY to another landblock XXYY' or things that a live server would never enable under normal circumstances. and even then, if i were developing w/ LogLevel.Debug, i would expect to see those messages on the console. i would expect the console to just display any log messages that i have enabled, including debug